docs/troubleshooting.rst

```diff
@@ -71,3 +71,11 @@ Make repository::
 
 Also, make sure the user that runs uwsgi can access /var/run/uwsgi and can
 write in the uwsgi directory.
+
+Unexpected database encoding sql_ascii
+======================================
+
```

Comment:
If possible I would add the exact message returned by postgresql. So that looking for the solution would come up in the search engine.

Also it is possible other encoding would raise a similar error.